### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by g-ortuno**
Basic hello world app
Please review @jyasskin @scheib @reillyeon

device-2015-03-02-182318

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,10 @@ proguard/
 
 # Log Files
 *.log
+
+# Idea files
+/.idea/workspace.xml
+/.idea/libraries
+
+# Mac Files
+.DS_Store
```
Comment (jyasskin) [inline .gitignore]: Try to keep files ending with a newline. There may be a setting in Android Studio or whichever editor you're using to do this.

File: app/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
+# By default, the flags in this file are appended to flags specified
+# in /usr/local/google/home/ortuno/android-sdk-linux/tools/proguard/proguard-android.txt
```
Comment (jyasskin) [inline app/proguard-rules.pro]: It'd be best not to check in files referring to anyone's local filesystem. (Too bad Android generates these to do this by default.) Maybe change this one to "$ANDROID_SDK/tools/proguard/proguard-android.txt"?

File: gradlew [hunk 3]
```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```
Comment (jyasskin) [inline gradlew]: Ugh. Android--. Why can't this be part of the SDK? :-P (not your fault; no need to change it)

**Comment by jyasskin** [comment]
LGTM other than the above comments.

File: .gitignore (hunk 1 above)
Comment (g-ortuno) [inline .gitignore]: Done.

File: app/proguard-rules.pro (hunk 2 above)
Comment (g-ortuno) [inline app/proguard-rules.pro]: Done.

**Comment by scheib** [comment]
LGTM Discussed verbally some points, and suggested this be moved to a src/ directory in a follow up patch.

**Pull request merged**